Formula/git-machete.rb

```diff
@@ -43,5 +45,7 @@ def install
     (testpath/".git/machete").write "main\n  develop"
     expected_output = "  main\n  |\n  | Other commit\n  o-develop *\n"
     assert_equal expected_output, shell_output("git machete status --list-commits")
+
+    assert_match /GIT-MACHETE\(1\)/, shell_output("git machete --help | head -1")
```

Comment:
`git PLUGIN --help` calls `man git-PLUGIN` under the hood (and not `git-PLUGIN --help`, as with everything else one might pass to `git PLUGIN`)